frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -16,91 +18,69 @@ export interface EmptySProps {
-              <Link className="pf-c-button pf-m-secondary" to="/import?importType=git">
-                Import from Git
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Deploy Image</CardHeader>
-            <CardBody>Deploy an existing image from an image registry or image stream tag</CardBody>
-            <CardFooter>
-              <Link
-                className="pf-c-button pf-m-secondary"
-                to={`/deploy-image?preselected-ns=${activeNamespace}`}
-              >
-                Deploy Image
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Browse Catalog</CardHeader>
-            <CardBody>Browse the catalog to discover, deploy and connect to services</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog">
-                Browse Catalog
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import from Dockerfile</CardHeader>
-            <CardBody>Import your Dockerfile from your git repo to be built & deployed</CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/import?importType=docker">
-                Import from Dockerfile
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Import YAML</CardHeader>
-            <CardBody>Create or replace resources from their YAML or JSON definitions.</CardBody>
-            <CardFooter>
-              <Link
-                className="pf-c-button pf-m-secondary"
-                to={formatNamespacedRouteForResource('import', activeNamespace)}
-              >
-                Import YAML
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-        <GridItem sm={6} md={6} lg={4}>
-          <Card className="odc-empty-state__card">
-            <CardHeader>Add Database</CardHeader>
-            <CardBody>
-              Browse the catalog to discover database services to add to your application
-            </CardBody>
-            <CardFooter>
-              <Link className="pf-c-button pf-m-secondary" to="/catalog?category=databases">
-                Add Database
-              </Link>
-            </CardFooter>
-          </Card>
-        </GridItem>
-      </Grid>
-    </div>
-  </React.Fragment>
-);
+const ODCEmptyState: React.FC<Props> = ({ title, activeNamespace }) => {
+  const navigateTo = (e: Event, url: string) => {
```

Comment:
This function should be moved out of this closure.

Reply:
Done.